ui/src/components/Welcome/WelcomeSlideOne.vue

```diff
@@ -2,11 +2,12 @@
     <div :class="['welcome-slide-one-wrapper', visible ? 'visible' : 'hidden',
     ]">
         <div class="welcome-text">
-            <h1 data-test="welcome-slide-one-title">Welcome to <br />OpenNMS&nbsp;Cloud!</h1>
-            <p class="margin-bottom">Our mission is to make monitoring just&nbsp;happen.</p>
-            <p class="margin-bottom">To start monitoring, your network must include at least one OpenNMS Minion for
-                distributed network monitoring. We'll walk you through that setup, which takes only a
-                few&nbsp;minutes.</p>
+            <h1 data-test="welcome-slide-one-title">Welcome to <br />OpenNMS&nbsp;Cloud</h1>
+            <p class="margin-bottom">To start monitoring, you need to install our Minion on your system. This lightweight,
+                secure collector monitors and communicates with the devices on your network and shares collected data with
+                OpenNMS Cloud. The Minion connects your network with OpenNMS&nbsp;Cloud. </p>
```

Comment:
```suggestion
and connects your network with OpenNMS&nbsp;Cloud. </p>
```
Tightened it up from the original.